goldstone/core/models.py

```diff
@@ -2634,3 +2634,38 @@ def integration(cls):
     def resourcetype(cls):
 
         return "routers"
+
+
+class EventQueryDef(PolyResource):
+
+    """ This model defines how events get generated from parsing
+        queries and log indices """
+
+    uuid = UUIDField(version=1, auto=True, primary_key=True)
```

Comment:
review PolyResource, and Django Polymorphic docs.  You should not need to declare many of these fields as they are inherited from PolyResource.